xml_converter/src/xml_converter.cpp

```diff
@@ -488,34 +516,61 @@ void process_data(
 // receive.
 //
 // Example usage
-//   ./xml_converter --input-path ../packs/marker_pack --output-path ../output_packs
-//   ./xml_converter --input-path ../packs/* --output-path ../output_packs
+//   ./xml_converter --input-taco-paths ../packs/marker_pack --output-split-waypoint-path ../output_packs
```

Comment:
`--input-taco-paths` should be changed to match the if statement in line 535.

Reply:
This is a valid critique. Overall it is a minor detail, I will fix it in a future pr instead of trying to rebase other commits for the sake of a comment.